CHANGELOG.md

```diff
@@ -1,11 +1,42 @@
 # Change Log
 
 ## v0.3.18
-- [core] Fix `@theia/core/lib/node/debug#DEBUG_MODE` flag to correctly detect when the runtime is inspected/debugged
-- [core] Add a preference to define how to handle application exit.
-- [core] Add a way to prevent application exit from extensions.
-- [core] Prevent application exit if some editors are dirty.
+- [core] added a preference to define how to handle application exit
+- [core] added a way to prevent application exit from extensions
+- [core] added functionality to prevent application exit if some editors are dirty
+- [core] allowed the ability to scope bindings per connection
+- [core] fixed `@theia/core/lib/node/debug#DEBUG_MODE` flag to correctly detect when the runtime is inspected/debugged
+- [cpp] fixed clangd being prematurely started when a build config is active
+- [electron] implemented HTTP-based authentication for Git
+- [electron] updated Electron to `0.2.14`
```

Comment:
`0.2.14` -> `2.0.14`

https://github.com/theia-ide/theia/blob/46973a4e81c5bbbc1705a827a0edea72ba6e3abd/packages/core/package.json#L23

Reply:
thanks @kittaakos for pointing out the error

Reply:
Good catch; I will update the changelog in place, as part of the release commit, to fix this.